vivisect/parsers/pe.py

```diff
@@ -115,11 +115,15 @@ def loadPeIntoWorkspace(vw, pe, filename=None, baseaddr=None):
     if fvivname is None:
         fvivname = "pe_%.8x" % baseaddr
 
-    fhash = "unknown hash"
-    if os.path.exists(filename):
-        fhash = v_parsers.md5File(filename)
+    # grab the file bytes for hashing
+    pe.fd.seek(0)
+    bytez = pe.fd.read()
+    fhash = v_parsers.md5Bytes(bytez)
+    sha256 = v_parsers.sha256Bytes(bytez)
 
+    # create the file and 
```

Comment:
Just to be persnickety, can you finish the comment?

Reply:
yup :)